cpp/tests/groupby/sort/type_info.hpp

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
This looks redundant with what is already in the hash groupby tests. Consolidate.